jobs/kpi-forecasting/kpi_forecasting/metric_hub.py

```diff
@@ -52,17 +59,32 @@ def __post_init__(self) -> None:
             "\n", "\n" + " " * 19
         )
 
+        # Add query snippets for segments
+        if self.segments:
+            segment_select_query = []
+            for alias, sql in self.segments.items():
+                segment_select_query.append(f"  {sql} AS {alias},")
+            self.segment_select_query = ",\n    ".join(segment_select_query)
+            self.segment_groupby_query = ",\n ,".join(self.segments.keys())
+        else:
+            self.segment_select_query = ""
+            self.segment_groupby_query = ""
```

Comment:
Another nit-picky pythonic question, should these 2 assignment go between lines 62 and 63 (i.e., outside the if/else) and then you can remove the 'else' altogether? I honestly don't know which is "more correct".

Reply:
I used the `else` to avoid those variables being assigned twice for cases where the `if` condition is satisfied. I'm also not sure which is more correct, and I am curious if there is a settled preference for this (couldn't find anything in my brief googling about it).